### DISCUSSION TITLE
v19.0.1

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1033] [General]
## Releases Notes for 19.0.1
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v19.0.1/draw.io-19.0.1-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v19.0.1/draw.io-19.0.1-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v19.0.1/draw.io-universal-19.0.1.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v19.0.1/drawio-amd64-19.0.1.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v19.0.1/drawio-x86_64-19.0.1.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v19.0.1/drawio-x86_64-19.0.1.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Switches to new code signing certificate on Windows. It all show a security prompt on Windows until a certain number of users have installed the app.
- Updates to [draw.io core 19.0.1](https://github.com/jgraph/drawio/blob/v19.0.1/ChangeLog). All changes from 19.0.1 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v19.0.1'>v19.0.1</a>.</em>

**Comment by u01jmg3** [comment]
Thanks for this - is `draw.io-x64-19.0.1.exe` no longer available?

**Reply by davidjgraph** [reply]
No, is that an issue?

**Reply by u01jmg3** [reply]
It's the download I would normally grab. Which one do you recommend instead?

**Reply by davidjgraph** [reply]
It's not signed. The downloads are linked at the top of the release notes.
